src/slices/AccountSlice.ts

```diff
@@ -94,19 +101,30 @@ export const getBalances = createAsyncThunk(
     let ohmV2Balance = BigNumber.from("0");
     let sohmV2Balance = BigNumber.from("0");
     let wsohmBalance = BigNumber.from("0");
+    let wsohmOnEthereum = BigNumber.from("0");
+    let wsohmOnAvax = BigNumber.from("0");
+    let wsohmOnArbitrum = BigNumber.from("0");
     let poolBalance = BigNumber.from("0");
     let fsohmBalance = BigNumber.from(0);
     let fgohmBalance = BigNumber.from(0);
     let fgOHMAsfsOHMBalance = BigNumber.from(0);
     let fiatDaowsohmBalance = BigNumber.from("0");
 
+    const ethProvider = NodeHelper.getMainnetStaticProvider();
```

Comment:
can you use this `ethProvider` to pull the other mainnet balances as well? (OHM, sOHM, etc). The main reason I'm leaning this way is so that your ohm & sOHM MAINNET balances are properly queried even if you load the app on another chain.

Reply:
yesirr